src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2.tsx

```diff
@@ -23,6 +21,16 @@ export const ArtworkSidebar2: React.FC<ArtworkSidebarProps> = props => {
   return (
     <Flex flexDirection="column">
       <ArtworkSidebar2ArtistsFragmentContainer artwork={artwork} />
+      <Flex flexDirection="row">
+        <Text color="black60" variant="lg-display" fontStyle="italic">
+          {artwork.title}
+        </Text>
+        <Text color="black60" variant="lg-display">
+          , {artwork.date}
+        </Text>
```

Comment:
@gkartalis I think we'll likely want to put these pieces of text in the same `<Text>` or us `as="span"` or find some other way of avoiding awkward formatting for long titles.

We might actually just want to steal what's in the current component verbatim since it seems to handle a few edge cases already: https://github.com/artsy/force/blob/470b6a3189f9592f4e5298687088e095f432b999/src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarTitleInfo.tsx#L20-L23

Example: http://localhost:5000/artwork/euan-roberts-green-balenciaga-pink-tiles

<img width="429" alt="Screen Shot 2022-08-31 at 15 22 30" src="https://user-images.githubusercontent.com/5361806/187688726-0f1ebd4a-82be-4313-b12b-925a8d0bb392.png">

Reply:
good catch